reaper/dicom_reaper.py

```diff
@@ -34,7 +34,7 @@ def instrument_query(self):
         if scu_series is None:
             return None
         for series in scu_series:
-            if series['NumberOfSeriesRelatedInstances'] is None:
+            if not series['NumberOfSeriesRelatedInstances']:
```

Comment:
This will also catch if `NumberOfSeriesRelatedInstances == 0`, which is probably ok.